binrz/man/rz-asm.1

```diff
@@ -1,108 +1,78 @@
 .It Fl a Ar arch
-Set architecture plugin
+Set architecture to assemble/disassemble (see -L)
 .It Fl A
-Show analysis information of given hexpair string
+Show analysis information from given hexpairs
 .It Fl b Ar bits
-Set architecture bits
+Set CPU register size (8, 16, 32, 64) (RZ_ASM_BITS)
 .It Fl B
-Binary input/output (\-l is mandatory for binary input)
-.It Fl c Ar cpu
-Select specific CPU (depends on \-a arch)
+Binary input/output (-l is mandatory for binary input)
+.It Fl c Ar CPU
+Select specific CPU (depends on the arch)
 .It Fl C
 Output in C format
-.It Fl d
-Disassemble hexpair bytes. rz-asm \-d 9090
-.It Fl D
-Disassemble showing hexpair and opcode
+.It Fl d, D
+Disassemble from hexpair bytes (-D show hexpairs)
 .It Fl e
-Use big endian (or swap endianness if used more than once)
+Use big endian instead of little endian
+.It Fl I
+Display lifted RzIL code (same input as in -d, IL is also validated)
 .It Fl E
-Output disassembled instructions in ESIL format.
-.It Fl f
-Read data from file instead of ARG.
+Display ESIL expression (same input as in -d).
+.It Fl f Ar file
+Read data from file.
 .It Fl F Ar in:out
 Specify input and/or output filters (att2intel, x86.pseudo, ...)
-.It Fl h
-Show usage help message.
-.It Fl hh
-Show long help message including supported assembler directives
-.It Fl l Ar int
+.It Fl h, hh
+Show this help, -hh for long.
+.It Fl i Ar len
+Ignore N bytes of the input buffer
+.It Fl j
+Output in JSON format
+.It Fl k Ar kernel
+Select operating system (linux, windows, darwin, ..)
+.It Fl l Ar len
 Input/Output length
-.It Fl i Ar int
-Ignore/skip N bytes from the beginning of the input buffer
 .It Fl L
-List supported asm plugins
-.It Fl o Ar offset
-Offset of the opcode to assemble (default is 0)
-.It Fl @ Ar offset
-Alias for -o
-.It Fl O Ar ofile
-output to file, for example 'rz-asm \-BF a a.asm'
+List asm plugins: (a=asm, d=disasm, A=analyze, e=ESIL)
+.It Fl o, @ Ar addr
+Set start address for code (default 0)
+.It Fl O Ar file
+Output file name (rz-asm -Bf a.asm -O a)
+.It Fl p
+Run SPP over input for assembly
+.It Fl q
+Quiet mode
 .It Fl r
-Show output in rizin script
+Output in rizin commands
 .It Fl s Ar syntax
-Select syntax output (intel, att)
-.It Fl w
-Describe opcode (whats op)
+Select syntax (intel, att)
+.It Fl v
+Show version number
 .It Fl x
-Use hex dwords instead of hexpairs in the assembler output
-.It Fl q
-Quiet output (handy for -L, -v, ...)
-.El
-.Sh different than filename
-.Pp
-.Bl -tag -width Fl
-.It Fl .intel_syntax
-Use intel syntax rather than att:w
-.It Fl .att_syntax
-Use ATT syntax rather than flu :w
-dentify the region fof aslerrger
-.It Fl string
-dentify the region fof aslerrger
-.It Fl .align
-.It Fl .arch
-.It Fl .arm
-.It Fl .ascii
-.It Fl .bits
-.It Fl .data
-.It Fl .equ
-.It Fl .fill
-.It Fl .glob
-.It Fl .hex
-.It Fl .int16
-.It Fl .int32
-.It Fl .int64
-.It Fl .kernel
-.It Fl .offset
-.It Fl .org
-.It Fl .os
-.It Fl .short
-.It Fl .text
-.It Fl .thumb
```

Comment:
Why removing these? Aren't we supporting these anymore?

Reply:
i guess they are missing in the help